jbmc/src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -157,6 +157,11 @@ void java_bytecode_languaget::get_language_options(const cmdlinet &cmd)
   else
     java_cp_include_files=".*";
 
+  if(cmd.isset("nondet-static"))
+  {
+    nondet_static = true;
```

Comment:
`generate_nondet_static_initializers`?

Reply:
There are two reasons for this name - first, it copies the name of the flag itself which seems to be often the case in this function (it makes it easier to see the connection then?) and second, this option does not only generate nondet staic initializers. It nondet initializes everything in CPROVER_init and also goes over static initializers and uses nondet there.